MidasCivil_Adapter/MidasCivilAdapter.cs

```diff
@@ -109,7 +114,7 @@ public static bool IsApplicationRunning()
 
         private List<string> m_midasText;
         private string m_directory;
-        public string m_midasCivilVersion { get; protected set; }
+        public string m_midasCivilVersion { get; protected set; } = "";
```

Comment:
There's a line in the `Execute` commands that sets a default value of 8.8.1 - can we update this to 9.4.0 please.

You can see the different versions here: https://github.com/BHoM/MidasCivil_Toolkit/wiki/MidasCivil-Specific-Adapter-Features#midascivil-versions

Reply:
Updated to latest version of MidasCivil, 9.4.0.